frontends/infinite-corridor/src/test-utils/testing_init.ts

```diff
@@ -15,7 +10,6 @@ global.SETTINGS = _createSettings()
 afterEach(function() {
   const node = document.querySelector("#integration_test_div")
   if (node) {
-    // @ts-expect-error we need to set SETTINGS for all tests
     ReactDOM.unmountComponentAtNode(node)
   }
   document.body.innerHTML = ""
```

Comment:
I think we can get rid of this all together. RTL cleans up after itself automatically in testing environments that support `afterEach`: https://testing-library.com/docs/react-testing-library/api/#cleanup

I also think it would be good to combine this file and `setupJest` into one.